### PR TITLE
f-vue-icons@3.10.0 - Add narrow visa icon

### PR DESCRIPTION
**Pull request by FayeCarter**
### Added
- `PaymentVisaTransparentNarrowIcon` icon.
visa

### CODE REVIEW

**Review by kevinrodrigues — DISMISSED** [review]

**Review by charliearlie — DISMISSED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -2376,10 +2376,10 @@
     axios "0.21.1"
     axios-mock-adapter "1.19.0"
 
-"@justeat/f-icons@4.9.0":
-  version "4.9.0"
-  resolved "https://registry.yarnpkg.com/@justeat/f-icons/-/f-icons-4.9.0.tgz#c0a901996311ad97300498885233f0ed6f47fdc4"
-  integrity sha512-8elkZrUaVygcvOxep+6qxlJP+jabXAFOrAw6k09z4Xkd9ABbpPZLZeI2sBHItCZb6OUTrBu3c5pV2V4iw0Q24A==
+"@justeat/f-icons@4.10.0":
+  version "4.10.0"
+  resolved "https://registry.npmjs.org/@justeat/f-icons/-/f-icons-4.10.0.tgz#59738afbb1bec54388270521ae8f6de0412d74fb"
```
Comment (zlatinivanov-je) [inline yarn.lock:2381]: does it matter if NPM was used?

**Review by zlatinivanov-je — DISMISSED** [review]
Just one question other than that LGTM

File: yarn.lock (hunk 1 above)
Comment (FayeCarter) [reply yarn.lock:2381]: Oh I hadn't spotted, I'll update now, thanks!

Comment (ashleynolan) [reply yarn.lock:2381]: Yep – this should be updated

**Review by ashleynolan — CHANGES_REQUESTED** [review]

**Review by BanksySan — APPROVED** [review]

**Review by ashleynolan — APPROVED** [review]

**Pull request merged**